### DISCUSSION TITLE
[vite] page reload error: module is not definined

### DISCUSSION THREAD

**Discussion by 3200pro** [sanity-io/sanity#3789] [Sanity Studio Support]
Hi Everyone,

I'm using svelteKit for the frontend. When I try to run dev I get the following error:

[vite] page reload studio/.sanity/runtime/index.html

```
module is not defined

ReferenceError: module is not defined
    at /studio/node_modules/react-fast-compare/index.js:118:1
    at instantiateModule (file:///.../dev/node_modules/vite/dist/node/chunks/dep-4da11a5e.js:53445:15)
```

Any idea where I messed up? Thanks in advance:)

**Comment by rexxars** [comment]
Could you create a reproducible test case? I'm not sure quite how you are integrating sveltekit and Sanity.

**Reply by 3200pro** [reply]
Hello @rexxars,

To simplify troubleshooting, I've created a public repo that is a skeleton build of SvelteKit + Sanity V3. I haven't hooked up the actual studio content at all yet, aside from a few very basic queries at `+page.server.js` (these are not added to the actual `page.svelte` page yet due to the error)

https://github.com/3200pro/sveltekit_sanityV3_testing

The error I get is: 
```
module is not defined
ReferenceError: module is not defined
    at /studio/node_modules/react-fast-compare/index.js:118:1
    at instantiateModule (file:///../node_modules/vite/dist/node/chunks/dep-61d2428a.js:55965:15
```

Let me know if you need anything else:)

**Comment by 3200creative** [comment]
Has anyone else run into this yet? I still don't have an idea of why this module is even getting picked up by vite.
